### ISSUE TITLE
Remmina SNAP can't still access keyring to save passwords

### ISSUE THREAD

**Issue by giox069** [FreeRDP/Remmina#1404]
On some distributions, verified on Ubuntu 17.04, 17.10 and 18.04, the remmina SNAP version is unable to access the GNOME keyring. Remmina SNAP also crashes under ubuntu 17.10 and 18.04.
On other distributions, verified on Debian 9 Stretch, remmina SNAP is able to access the gnome keyring and saving passwords without problems.
Snap versions used on all four distributions, working and not working, is 2.29.4.2

Here is the console output for Ubuntu 17.10 when launching remmina:
```
** (remmina:2161): CRITICAL **: secret_service_load_collections_sync: assertion 'paths != NULL' failed

** (remmina:2161): CRITICAL **: secret_service_read_alias_dbus_path_sync: assertion 'SECRET_IS_SERVICE (self)' failed
StatusNotifier/Appindicator support: your desktop does support it and libappindicator is compiled in remmina. Good!

(remmina:2161): Gtk-WARNING **: gtk_menu_attach_to_widget(): menu already attached to GtkMenuItem

(remmina:2161): Gtk-WARNING **: Error loading theme icon 'list-add' for stock: Icon 'list-add' not present in theme Adwaita

(remmina:2161): Gtk-WARNING **: Error loading theme icon 'edit-clear' for stock: 
Segmentation fault (core dumped)

```
This crash is also closely related to issue #1288
This crash also happens under ubuntu 18.04, but not under ubuntu gnome/xubuntu 17.04.

A more interesting error under Ubuntu Gnome 17.04 appears when saving the password:
```
** Message: Remote error from secret service: org.freedesktop.DBus.Error.AccessDenied: An AppArmor policy prevents this sender from sending this message to this recipient; type="method_call", sender=":1.93" (uid=1000 pid=12223 comm="remmina ") interface="org.freedesktop.Secret.Service" member="SearchItems" error name="(unset)" requested_reply="0" destination=":1.19" (uid=1000 pid=1214 comm="/usr/bin/gnome-keyring-daemon --daemonize --login ")

```

**Comment by giox069** [comment]
It seems that executing
```
sudo snap connect remmina:password-manager-service  :password-manager-service
```
and then rebooting, should be enough to fix it.
Please report here positive or negative results.

**Comment by alex-mashin** [comment]
It really helped, thank you very much.

**Comment by adback03** [comment]
I have verified that the fix in @giox069's comment solved the issue for me in Ubuntu 16.04.
